app/experimenter/experiments/bugzilla.py

```diff
@@ -38,13 +53,52 @@ def format_bug_body(experiment):
             ]
         )
         bug_body = experiment.BUGZILLA_PREF_TEMPLATE.format(
-            experiment=experiment, variants=variants_body
+            experiment=experiment,
+            variants=variants_body,
+            countries=countries,
+            locales=locales,
         )
 
     return bug_body
 
 
-def make_bugzilla_call(url, data):
+def format_update_body(experiment):
+    cf_tracking = "cf_tracking_firefox{}".format(
+        get_firefox_major_version(experiment.firefox_version)
+    )
+    summary = "[Experiment] {experiment_name} Fx {version} {channel}".format(
+        experiment_name=experiment,
+        version=experiment.firefox_version,
+        channel=experiment.firefox_channel,
+    )
+    return {
+        "summary": summary,
+        "cf_user_story": format_bug_body(experiment),
+        "whiteboard": experiment.STATUS_SHIP_LABEL,
+        cf_tracking: "?",
+    }
+
+
+def update_experiment_bug(experiment):
+    body = format_update_body(experiment)
+    make_bugzilla_put_call(
+        settings.BUGZILLA_UPDATE_URL.format(id=experiment.bugzilla_id), body
+    )
+
+
+def make_bugzilla_put_call(url, data):
+    try:
+        response = requests.put(url, data)
+        return json.loads(response.content)
```

Comment:
followed what was done with existing rest calls but just curious the diff between ``` response.json()``` vs ``` json.loads(response.content)``` <- might be a silly question 🙈

Reply:
@tiftran Good catch! @peterbe also pointed this out a little while ago and filed this:

https://github.com/mozilla/experimenter/issues/982

so if you want to switch that here we can also close 982, or we could leave it and do it separately, up to you.